src/pages/PublicModule/PublicModule.tsx

```diff
@@ -132,7 +132,7 @@ class PublicModule extends Component<PublicModuleProps, MyModuleState> {
           <Card.Body>
             <Card.Title>{module.name}</Card.Title>
             <Card.Text style={{height: 105, textOverflow: 'ellipsis', overflow: 'hidden'}}>
-              {module.description.substring(0, 150)}
+              {module.description}
```

Comment:
The startModule method in this file still calls
 `const userModule = await startUserModule(String(this.state.module.specialCode));` which is directly referencing the specialCode.